designs/usb-accel.stanza

```diff
@@ -0,0 +1,47 @@
+#use-added-syntax(jitx)
+defpackage ocdb/designs/usb-accel :
+
+  import core
+  import collections
+  import math
+  import jitx
+  import jitx/commands
+
+  import ocdb/bundles
+  import ocdb/connects
+  import ocdb/defaults
+  import ocdb/design-vars
+  import ocdb/generator-utils
+  import ocdb/generic-components
+  import ocdb/property-structs
+  import ocdb/checks
+  import ocdb/micro-controllers
+
+val BOARD-SHAPE = RoundedRectangle(50.0, 50.0, 0.25)
```

Comment:
Can you please describe what you expect here for the radius at the corners? 0.25 or 12.5?

Reply:
0.25